packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -275,7 +280,8 @@ export class ResourceService {
       data: { ...USER_RESOURCE_ROLE, resourceId: resource.id },
     });
 
-    await this.entityService.createDefaultEntities(resource.id, user);
+    requireAuthenticationEntity &&
+      (await this.entityService.createDefaultEntities(resource.id, user));
```

Comment:
What will happen if requireAuthenticationEntity is false or falsy (undefined)?

Reply:
@abrl91 add default value = true